vllm/model_executor/layers/linear.py

```
@@ -1229,6 +1284,35 @@ def weight_loader(
+                        "total": (
+                            (
+                                self.loaded_total_num_heads
+                                + self.loaded_total_num_kv_heads
+                            )
+                            * self.head_size
+                            + self.loaded_total_num_kv_heads * self.v_head_size,
+                            0,
+                        ),
+                    }
+
+                    shard_size, shard_offset = adjust_bitsandbytes_4bit_shard(
```

Ruff (0.16.2)

[error] Undefined name `adjust_bitsandbytes_4bit_shard`

(F821)